camel-example-spring-boot-load-balancer-eip/README.adoc

```diff
@@ -0,0 +1,31 @@
+== Camel Example Spring Boot
+
+This example shows several examples of Load Balancer EIP with Apache Camel application using Spring Boot.
+
+1. Round-robin - The exchanges are selected from in a round robin fashion. This is a well known and classic policy, which spreads the load evenly.
```

Comment:
I think it should be only "in" and not "from in"

Reply:
Description comes from https://camel.apache.org/components/latest/eips/loadBalance-eip.html
Do you want changes in both places?

Reply:
Sounds good

Reply:
All requested changes are done, kindly verify.